hybrasyl/Dialogs.cs

```diff
@@ -437,14 +458,30 @@ public void HandleResponse(WorldObject invoker, int optionSelected, WorldObject
                     Associate = Sequence.Associate;
                 else
                     Associate = associateOverride;
-
                 // Note that client is 1-indexed for responses
+                // If we have a JumpDialog, handle that first
+
                 if (Options[optionSelected - 1].JumpDialog != null)
                 {
                     // Use jump dialog first
                     Options[optionSelected - 1].JumpDialog.ShowTo(invoker as User, Associate as VisibleObject);
                     return;
                 }
+
+                // If the response is a sequence, start it
+                if (Options[optionSelected - 1].overrideSequence != null)

```

Comment:
This looks hacky, what's the reason for going back 1?

Reply:
The client requires it.